<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newGameDialog</class>
 <widget class="QWidget" name="newGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Players</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QTreeView" name="playerList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="addPlayerButton">
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="removePlayerButton">
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Map</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Neutral planets:</string>
        </property>
        <property name="buddy">
         <cstring>neutralPlanetsSB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="KIntSpinBox" name="neutralPlanetsSB">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Width:</string>
        </property>
        <property name="buddy">
         <cstring>widthSB</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KIntSpinBox" name="widthSB">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Height:</string>
        </property>
        <property name="buddy">
         <cstring>heightSB</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="KIntSpinBox" name="heightSB">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2" colspan="3">
       <widget class="MiniMapView" name="map" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Production:</string>
        </property>
        <property name="buddy">
         <cstring>neutralPlanetsSB</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="ProductionSB">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="KillPercentageSB">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Kill Percentage:</string>
        </property>
        <property name="buddy">
         <cstring>neutralPlanetsSB</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="randomizeMap">
        <property name="text">
         <string>&amp;Randomize</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Owner</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="OwnerCB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="BlindMapCB">
        <property name="text">
         <string>Blind Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CumulativeProductionCB">
        <property name="text">
         <string>Cumulative Production</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ProductionAfterConquereCB">
        <property name="text">
         <string>Production After Capture</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Neutrals</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="NeutralsShowShipsCB">
           <property name="text">
            <string>Show Ships</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="NeutralsShowStatsCB">
           <property name="text">
            <string>Show Stats</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Production</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="NeutralsProductionSB">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>MiniMapView</class>
   <extends>QWidget</extends>
   <header>minimapview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
